components/server/ee/src/workspace/workspace-factory.ts

```diff
@@ -363,14 +363,17 @@ export class WorkspaceFactoryEE extends WorkspaceFactory {
             const billingTier = await this.entitlementService.getBillingTier(user);
             const userTeams = await this.teamDB.findTeamsByUser(user.id);
             // this allows to control user`s PVC feature flag via ConfigCat
-            if (
-                await getExperimentsClientForBackend().getValueAsync("user_pvc", false, {
-                    user,
-                    teams: userTeams,
```

Comment:
IMO we should just remove `teams` here, without the loop. It does not look like we use the team inheritance here.